tools/common/Frameworks.cs

```diff
@@ -276,7 +276,6 @@ public Framework Find (string framework)
 
 					{ "ScreenCaptureKit", "ScreenCaptureKit", 12,3 },
 
-					{ "BackgroundAssets", "BackgroundAssets", 13,0 },
```

Comment:
Leave it for macOS, not for the other platforms. Looks to me we are doing a step fwd and 2 steps back with this.